skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConverters.scala

```diff
@@ -89,6 +91,54 @@ object ConsumerConverters {
     }
   }
 
+  implicit class RebalanceListener1Ops[F[_]](val self: RebalanceListener1[F]) extends AnyVal {
+
+    def asJava(consumer: ConsumerJ[_, _])(implicit
+      F: Concurrent[F],
+      toTry: ToTry[F],
+    ): RebalanceListenerJ = {
+
+      def onPartitions(
+        partitions: CollectionJ[TopicPartitionJ],
+        call: Nes[TopicPartition] => RebalanceCallback[F, Unit]
+      ): Unit = {
+        // If you're thinking about deriving ToTry timeout based on ConsumerConfig.maxPollInterval
+        // please have a look on https://github.com/evolution-gaming/skafka/issues/125
+        val result = partitions
+          .asScala
+          .toList
+          .traverse { _.asScala[F] }
+          .map { partitions =>
+            partitions
+              .toSortedSet
+              .toNes
+          }
+          .toTry
+          .flatMap { nesOpt =>
+            nesOpt.map {
```

Comment:
```
map { ... }.getOrElse(().pure[Try])
```
could be replaced with `foldMapM` or `foldMapA`, afaik.

Reply:
good idea! thanks! fixed in 2822c5e0